lib/matplotlib/legend.py

```diff
@@ -185,6 +185,7 @@ def __init__(self, parent, handles, labels,
                  bbox_transform=None,  # transform for the bbox
                  frameon=None,  # draw frame
                  handler_map=None,
+                 uniform_size=None,  # set legend points to be a uniform size
```

Comment:
Okay. I noticed the keyword arguments for legend is also mentioned in figure.py. I'll add to that part as well.